tests/unistd/dup.c

```diff
@@ -29,7 +57,6 @@ int main() {
   printf("close(f1): %d\n", close(f));
   printf("close(f2): %d\n", close(f2));
   printf("close(f3): %d\n", close(f3));
-  printf("\n");
```

Comment:
Several tests have additional output that is useful when debugging them. For example, the test might print out addresses on the stack, which can change over builds. So it can be in the expected output. But those addresses are nice when debugging.

Additionally, node.js has a bug we work around with flushing output, and the workaround is to print an additional line of output at the end.